lib/Widget/dashboard/ADL_chart.dart

```diff
@@ -2,8 +2,62 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:syncfusion_flutter_gauges/gauges.dart';
 
-class ADLChart extends StatelessWidget {
-  Widget adlradialGauge(double score) {
+class ADLChart extends StatefulWidget {
+  @override
+  _ADLChartState createState() => _ADLChartState();
+}
+
+class _ADLChartState extends State<ADLChart> {
+  double score;
+  String topic;
+
+  Widget adlradialGaugecriteria(
```

Comment:
Consider using camelCase  For ex. `adlradialGaugecriteria` => `adlRadialGaugeCriteria` for more readibility

Reply:
Resolve in 143d0ec